src/sourmash/minhash.py

```diff
@@ -625,7 +625,7 @@ def set_abundances(self, values, clear=True):
 
             for h, v in values.items():
                 hashes.append(h)
-                abunds.append(v)
+                abunds.append(max(0, v))
```

Comment:
Hot take: I think this behavior is strange! We don't support negative abundances, and probably shouldn't support passing in negative abundances.

How about:
* 0 deletes hash
* < 0 raises `ValueError`
?

Reply:
Yes, more reasonable.

Ok, I have a question. Is it useful/supported to change specific abundances without dropping the rest? `set_abundance` can add to the current values but can't change/reduce.

Reply:
Yah, I don't know 😄 . We aren't really using abundances a lot at the moment so I don't have many good use cases in mind. To me, that suggests that any choice we make is going to be wrong... It's easy enough to add new methods or adjust this one down the road, so let's just make sure whatever you choose is well tested! Unless @luizirber has thoughts.

Reply:
hmmm... I was thinking of adding a `bool overwrite` flag so the user can overwrite some abundance values without clearing the rest, but, honestly, I have no idea about its importance. Let's just wait for a use case 😄

Reply:
:+1: on waiting for use case